client/helpers/files.ts

```diff
@@ -1,15 +1,15 @@
 import { CreateFileRequest, FileMetadataData, FileList } from '../../index';
 import {
-  BaseArgs,
-  DeleteArgs,
+  BaseReq,
+  DeleteReq,
```

Comment:
I renamed `Args` to `Req`. In new helpers, I'll also have a `Res` like we do in gRPC calls. This allows us to add new fields to responses without breaking clients.